frame/staking/src/pallet/mod.rs

```diff
@@ -24,8 +24,8 @@ use frame_support::{
 	dispatch::Codec,
 	pallet_prelude::*,
 	traits::{
-		Currency, CurrencyToVote, Defensive, DefensiveResult, DefensiveSaturating, EnsureOrigin,
-		EstimateNextNewSession, Get, LockIdentifier, LockableCurrency, OnUnbalanced, TryCollect,
+		fungibles, fungibles::Lockable, Currency, CurrencyToVote, Defensive, DefensiveResult,
```

Comment:
We should be doing one way or the other.
```suggestion
		fungibles, Currency, CurrencyToVote, Defensive, DefensiveResult,
```

Reply:
Thanks. For some reason i feel like i got some traut-related error when i tried this before. i'll try again and see

Reply:
Commiting this brought back the error I was talking about, so i have to revert.
<img width="1185" alt="image" src="https://user-images.githubusercontent.com/6564482/206429406-b35ff59e-6a32-430a-99b7-6d497a8b11db.png">